### PR TITLE
[REVIEW] Pin `cuda-python` to 11.7

### PR DESCRIPTION
**Pull request by galipremsagar**
Pinning to be inline with: https://github.com/rapidsai/integration/pull/495

### CODE REVIEW

**Review by ajschmidt8 — APPROVED** [review]

File: conda/environments/cudf_dev_cuda11.5.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ dependencies:
   - ipython
   - pandoc=<2.0.0
   - cudatoolkit=11.5
-  - cuda-python >=11.5,<12.0
+  - conda-forge::cuda-python >=11.5,<12.0
```
Comment (galipremsagar) [inline conda/environments/cudf_dev_cuda11.5.yml:41]:
```suggestion
  - cuda-python >=11.5,<12.0
```

File: conda/recipes/cudf/meta.yaml [hunk 2]
```diff
@@ -63,7 +63,7 @@ requirements:
     - packaging
     - cachetools
     - ptxcompiler  # [linux64]  # CUDA enhanced compatibility. See https://github.com/rapidsai/ptxcompiler
-    - cuda-python >=11.5,<12.0
+    - conda-forge::cuda-python >=11.5,<11.7.1
```
Comment (galipremsagar) [inline conda/recipes/cudf/meta.yaml:66]:
```suggestion
    - cuda-python >=11.5,<11.7.1
```

**Comment by galipremsagar** [comment]
@gpucibot merge

**Comment by galipremsagar** [comment]
closing in favor of : https://github.com/rapidsai/cudf/pull/11174

**Pull request closed without merging**